core/includes/config.inc

```diff
@@ -87,6 +92,11 @@ function config_clear($config_file, $option) {
  * @return mixed
  *   The contents of the requested config option. Returns NULL if the specified
  *   option was not found in the file at all.
+ *
+ * @see config_set()
+ * @see config_diff()
+ * @see config_clear()
+ * @see config_menu()
```

Comment:
Same here, I don't think `config_diff()` and `config_menu()` are necessary here.

Reply:
Agree.